### PR TITLE
Bugfix. Indentation inside string templates(#758)

### LINKED ISSUE #758: Diktat should check indentation inside string templates in {}
```kotlin
"Text: ${
expression
}"
```

`expression` should be indented one step more

### PR DESCRIPTION
**Pull request by aktsay6**
## What's done:
  * Added test
  * Fixed bug

This pull request closes #758 

## Actions checklist
* [ ] Implemented Rule, added Warnings
* [X] Added tests on checks
* [X] Added tests on fixers
* [ ] Updated diktat-analysis.yml
* [ ] Updated available-rules.md

### CODE REVIEW

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleWarnTest.kt [hunk 1]
```diff
@@ -606,5 +606,29 @@ class IndentationRuleWarnTest : LintTestBase(::IndentationRule) {
...
 
+    @Test
+    @Tag(WarningNames.WRONG_INDENTATION)
+    fun `should trigger on string templates starting with new line`() {
+        lintMethod(
+            """
+                |fun foo(some: String) {
+                |    fun bar() {
+                |        val a = "${'$'}{
+                |        expression
```
Comment (kentr0w) [inline diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleWarnTest.kt:617]: I corrected `expression` as it should be, but then it incorrectly detects the indentation of the bottom lines

**Review by kentr0w — CHANGES_REQUESTED** [review]

**Review by petertrr — APPROVED** [review]

**Review by akuleshov7 — APPROVED** [review]
lg for me, but please check comment from Denis

File: diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleWarnTest.kt (hunk 1 above)
Comment (aktsay6) [reply diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/IndentationRuleWarnTest.kt:617]: Fixed

**Review by kentr0w — APPROVED** [review]
lgtm

**Pull request merged**